src/Database/Postgres/Temp/Internal.hs

```diff
@@ -134,7 +134,7 @@ startWithLogger logger options mainDir stdOut stdErr = try $ flip onException (r
 
   logger InitDB
   initDBExitCode <- runProcessWith stdOut stdErr "initdb"
-      ("initdb --nosync -E UNICODE -A trust -D " ++ dataDir)
+      ("initdb -E UNICODE -A trust -D " ++ dataDir)
```

Comment:
I think --nosync allows faster startup, I wonder if we can expose this for users if they have 9.3 and above. I guess we should profile first to see if it is worth it. I'm fine removing it to improve compatibility.